src/Components/Movies.js

```diff
@@ -0,0 +1,55 @@
+import axios from "axios";
+import React from "react";
+
+class Movies extends React.Component {
+  state = { movies: [], moviePickedURL: "", moviePicked: {} };
+
+  seeMovies = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+      this.setState({ movies: res.data });
+    } catch (error) {
+      this.setState({ movies: [] });
+    }
+  };
+
+  componentDidMount() {
+    this.seeMovies();
+  }
+
+  pickAMovie = async (e) => {
+    this.setState({ moviePickedURL: e.target.value });
```

Comment:
If you are going to call it in the next line, I don't think you're going to need to keep track of it in your state